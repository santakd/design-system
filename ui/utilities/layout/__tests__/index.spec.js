/* eslint-env jest */
import React from 'react';
import { Magnet } from '../';
import createHelpers from '../../../../jest.helpers';

const { matchesMarkup } = createHelpers(__dirname);

it('renders top magnet', () =>
  matchesMarkup(<Magnet className="slds-has-top-magnet" />));

it('renders bottom magnet', () =>
  matchesMarkup(<Magnet className="slds-has-bottom-magnet" />));

it('renders touching magnets', () =>
  matchesMarkup(
    <div>
      <Magnet className="slds-has-bottom-magnet" />
      <Magnet className="slds-has-top-magnet" />
    </div>
  ));
