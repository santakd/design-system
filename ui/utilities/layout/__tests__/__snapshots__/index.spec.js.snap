// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders bottom magnet 1`] = `
<div
  className="slds-card slds-has-bottom-magnet"
>
  <div
    className="slds-p-around_medium"
  >
    My Component
  </div>
</div>
`;

exports[`renders top magnet 1`] = `
<div
  className="slds-card slds-has-top-magnet"
>
  <div
    className="slds-p-around_medium"
  >
    My Component
  </div>
</div>
`;

exports[`renders touching magnets 1`] = `
<div>
  <div
    className="slds-card slds-has-bottom-magnet"
  >
    <div
      className="slds-p-around_medium"
    >
      My Component
    </div>
  </div>
  <div
    className="slds-card slds-has-top-magnet"
  >
    <div
      className="slds-p-around_medium"
    >
      My Component
    </div>
  </div>
</div>
`;
